src/main/java/nl/tudelft/scrumbledore/Collision.java

```diff
@@ -6,6 +6,7 @@
  * @author Jesse Tilro
  *
  */
+@SuppressWarnings("PMD.TooManyMethods")
```

Comment:
Yes, code optimization eventually has to take place, but currently this would require an entire rewrite. Might be a good candidate to be planned in one of the upcoming weekends though.